tests/mo-timing-synchronization/OPS/content_001.xhtml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Basic Media Overlay Test</title>
+        <!-- <link rel="stylesheet" type="text/css" href="../css/base.css" /> -->
+	</head>
+	<body>
+
+        <p>Test passes (i.e., the Reading System implements Media Overlays) if</p>
+
+        <ol>
+            <li>The Reading System provides the user with a way to turn text reading on</li>
+            <li>If turned out, the extract on the next page is read aloud by the Reading System</li>
+            <li>The text is highlighed when its text is read; at first sentence by sentence, then paragraphs by paragraphs.</li>
```

Comment:
How about "The text is highlighted as its audio is played", without mentioning granularity (which might get confused with granularity *switching*, e.g. any given portion can be played word by word or paragraph by paragraph, seen only in laboratory experiments thus far but often discussed as a potential feature of MO).

Reply:
Yes, will do